RELEASE.md

```diff
@@ -0,0 +1,89 @@
+# Release management for protocol buffer specifications
+
+This repository primary provides two features:
+
+* Protobuf specifications for messages used within Sigstore.
+* Language clients for different ecosystems.
```

Comment:
Extreme nitpick: this repo is providing language bindings for the protobuf specifications, which are then used by clients in other repositories. Not sure if it's worth an edit, but the current language might be confusing to a (distant) future maintainer 🙂

Reply:
Yeah, so you would prefer s/clients/bindings/ ?

Reply:
Yeah, that SGTM!